client/src/screens/CourseRegistration/CourseRegistration.js

```diff
@@ -90,7 +72,62 @@ class CourseRegistration extends Component {
     }));
   }
 
+  // Returns the semesters displayed in the drop down
+  findSemesters(startYear) {
+    const today = new Date();
+    const currentYear = today.getFullYear();
+    let year = startYear;
+    let semesters = [];
+    for (year; year <= currentYear; year++) {
+      if (year === startYear) {
+        semesters.push('Fall ' + year.toString());
+      } else if (year === currentYear) {
+        semesters.push('Winter ' + year.toString());
+        if (today.getMonth() > 8) {
+          semesters.push('Fall ' + year.toString());
+        }
+      } else {
+        semesters.push('Winter ' + year.toString());
+        semesters.push('Fall ' + year.toString());
+      }
+    }
+    return semesters;
+  }
+
+  // TODO: remove because the student data should be passed down from login
+  fetchUserData() {
```

Comment:
it's an async call so it'd better to put async/await, otherwise u might get unexpected result.
```
fetchUserData = async() => {
    return await axios.get(
      'http://localhost:3001/users/getStudentData?studentID=999999999'
    );
  }
```